source/jasmine/index.ts

```diff
@@ -57,7 +57,7 @@ export function configure(
         defaultAssert!(a, m);
       }
     },
-    assertDeepEqual: (a, e) => expect(a).toEqual(e)
+    assertDeepEqual: (a, e) => expect(a).toStrictEqual(e)
```

Comment:
AFAICT, there is no `isStrictEqual` for Jasmine, so this will have to be reverted. And if the failing test no longer fails, that will have to be removed, too.

`assertDeepEqual` is only called if the matcher sees [non-arrays](https://github.com/cartant/rxjs-marbles/blob/75b646c92765e24f42119ba1e2855946ff391811/source/matcher.ts#L81) or if the config specifies that the [framework matcher should be used](https://github.com/cartant/rxjs-marbles/blob/75b646c92765e24f42119ba1e2855946ff391811/source/matcher.ts#L64) - which is the case for Jest.

Reply:
Hmm, that test case does fail when I run `yarn run test:jasmine`, just the way it does for Jest. Any idea why that is?

Reply:
I explained why in my comment. The test does not exercise the calling of that assertion. If you check the Jasmine docs and repo, you will see that the method does not exist.

Reply:
Fair enough! Reverting.